authenticator.js

```
@@ -0,0 +1,18 @@
+const jwt = require('jsonwebtoken');
+const logger = require('./logger');
+
+const config = require('./config');
+
+function authMiddleware(req, res, next) {
+    try {
+        if (!req.headers.authorization) throw Error;
```

Please throw the error with a message.